core/src/main/java/io/seata/core/constants/ConfigurationKeys.java

```diff
@@ -203,27 +201,27 @@
     /**
      * The constant STORE_DB_MIN_CONN.
      */
-    public static final String STORE_DB_MIN_CONN = STORE_DB_PREFIX + "min-conn";
+    public static final String STORE_DB_MIN_CONN = STORE_DB_PREFIX + "minConn";
 
     /**
      * The constant STORE_DB_MAX_CONN.
      */
-    public static final String STORE_DB_MAX_CONN = STORE_DB_PREFIX + "max-conn";
+    public static final String STORE_DB_MAX_CONN = STORE_DB_PREFIX + "maxConn";
 
     /**
      * The constant STORE_DB_LOG_QUERY_LIMIT.
      */
-    public static final String STORE_DB_LOG_QUERY_LIMIT = STORE_DB_PREFIX + "query-limit";
+    public static final String STORE_DB_LOG_QUERY_LIMIT = STORE_DB_PREFIX + "queryLimit";
 
     /**
      * The constant LOCK_DB_TABLE.
      */
-    public static final String LOCK_DB_TABLE = STORE_DB_PREFIX + "lock-table";
+    public static final String LOCK_DB_TABLE = STORE_DB_PREFIX + "lockTable";
 
     /**
      * The constant LOCK_DB_DEFAULT_TABLE.
      */
-    public static final String LOCK_DB_DEFAULT_TABLE = "lock_table";
+    public static final String LOCK_DB_DEFAULT_TABLE = "lockTable";
```

Comment:
lockTable-> lock_table
and move to DeafultValues

Reply:
great found.